crates/turbopack-core/src/output.rs

```diff
@@ -25,11 +25,7 @@ pub struct OutputAssets(Vec<Vc<Box<dyn OutputAsset>>>);
 impl OutputAssets {
     #[turbo_tasks::function]
     pub fn new(assets: Vec<Vc<Box<dyn OutputAsset>>>) -> Vc<Self> {
-        if assets.is_empty() {
-            OutputAssets::empty()
-        } else {
-            Vc::cell(assets)
-        }
+        Vc::cell(assets)
     }
 
     #[turbo_tasks::function]
```

Comment:
``` diff
-    #[turbo_tasks::function]
     pub fn empty() -> Vc<Self> {
-        Vc::cell(vec![])
+        Self::new(vec![])
     }
```

to make `OutputAssets::new(vec![])` == `OutputAssets::empty()`. They can return the same Vc

Reply:
Done.

Reply:
the empty fn need to be moved into a separate impl block for it for compile fine